plugin/src/main/java/co/brainly/onesky/OneSkyPlugin.kt

```diff
@@ -20,7 +20,8 @@ open class OneSkyPluginExtension(
     var apiKey: String = "",
     var apiSecret: String = "",
     var projectId: Int = -1,
-    var sourceStringFiles: List<String> = emptyList()
+    var sourceStringFiles: List<String> = emptyList(),
+    var sourcePath: String = "src/main/res"
```

Comment:
Could you please reflect this add-on in `README` file? 
It would be worth to mention that it has a default value and does not need to be set unless required.

Reply:
Hi @scana, no worries. That's a good point, already done